ERCS/erc-draft.md

```diff
@@ -0,0 +1,491 @@
+---
```

Comment:
```suggestion
---
eip: 7695
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.